tests/data/scripts/unit/test_clean_current_year.py

```diff
@@ -3,7 +3,7 @@
 import csv
 import pandas as pd
 
-from src.data.scripts import clean_and_pare_down_data_current_year
```

Comment:
This file no longer exists, and it wasn't actually doing anything useful - `clean_and_pare_down_data_all_years` (now renamed to `clean_and_split_data`) splits the data into the historic and latest CSVs used by the sub-sequent steps, so I updated this to test that instead.